runtime/v2/config.go

```diff
@@ -0,0 +1,59 @@
+package runtime
+
+import (
+	"strings"
+
+	"cosmossdk.io/core/server"
+	"cosmossdk.io/depinject"
+)
+
+// GlobalConfig is a recursive configuration map containing configuration
+// key-value pairs parsed from the configuration file, flags, or other
+// input sources.
+type GlobalConfig server.ConfigMap
```

Comment:
Is that wrapper really necessary?
I think it will resolve without it

Reply:
cc @kocubinski

Reply:
without the alias DI will err with `Duplicate Provision of server.ConfigMap`.  The reason for this is that `server.ConfigMap` is provided in module scope for each module (below), that's how they fetch only their config without other modules' configs being accessible.

```golang
func ProvideModuleScopedConfigMap(
	key depinject.ModuleKey,
	moduleConfigs ModuleConfigMaps,
) server.ConfigMap {
	return moduleConfigs[key.Name()]
}
```

so the options are:

1) alias `server.ConfigMap` to distinguish it in global scope or
2) alias `server.ConfigMap` to distinguish it in module scope

I chose (1) but (2) would also work.

Reply:
I see, thanks, that makes sense 👍